client/jetpack-connect/controller.js

```diff
@@ -282,7 +282,9 @@ export function plansLanding( context, next ) {
 
 	context.store.dispatch( setTitle( translate( 'Plans', { textOnly: true } ) ) );
 
-	analytics.tracks.recordEvent( 'calypso_plans_view' );
+	analytics.tracks.recordEvent( 'calypso_plans_view', {
+		small_screen: isSmallScreen(),
+	} );
```

Comment:
This event is fired anytime you view plans. This is likely the only event you need and you can use different event props for filtering:

* `siteidlabel` `jetpack || wpcom`. Careful, Atomic reports "jetpack"
* `browserdocumentlocation` Track flows you're interested in, like https://wordpress.com/jetpack/conect/plans/SITE_SLUG. Requires fuzzy matching, not sure if Hypothesis will support

If `browserdocumentlocation` won't work in hypothesis, we'll need to find another approach.

Reply:
How does this sound?

- `calypso_page_view` > `path: /jetpack/connect/store/:site`
- `calypso_plans_view` > `small_screen: 1`
- `wpcom_new_purchase_complete` > `service_slug: automattic`

Reply:
> calypso_abtest_start

I don't think you need event at all, hypothesis should take care of things.

Much of this isn't too relevant if dedicated events are added just for this test 🙂